src/data/electionSample.json

```diff
@@ -1,55 +1,580 @@
+          "party": "Constitution"
+        }
+      ],
+      "writeIns": false
+    },
+    {
+      "id": "state-assembly",
+      "type": "plurality",
+      "section": "54th District",
+      "title": "State Assembly",
+      "candidates": [
+        {
+          "id": "solis",
+          "name": "Andrea Solis",
+          "party": "Federalist"
+        },
+        {
+          "id": "keller",
+          "name": "Amos Keller",
+          "party": "People's"
+        },
+        {
+          "id": "rangel",
+          "name": "Davitra Rangel",
+          "party": "Liberty"
+        }
+      ],
+      "writeIns": false
+    },
+    {
+      "id": "county-commissioners",
+      "type": "plurality",
+      "section": "County",
+      "title": "County Commissioners",
+      "seats": 4,
+      "candidates": [
+        {
+          "id": "argent",
+          "name": "Camille Argent",
+          "party": "Federalist"
+        },
+        {
+          "id": "witherspoonsmithson",
+          "name": "Chloe Witherspoon-Smithson",
+          "party": "Federalist"
+        },
+        {
+          "id": "bainbridge",
+          "name": "Clayton Bainbridge",
+          "party": "Federalist"
+        },
+        {
+          "id": "hennessey",
+          "name": "Charlene Hennessey",
+          "party": "People's"
+        },
+        {
+          "id": "savoy",
+          "name": "Eric Savoy",
+          "party": "People's"
+        },
+        {
+          "id": "tawa",
+          "name": "Susan Tawa",
+          "party": "People's"
+        },
+        {
+          "id": "tawa-mary",
+          "name": "Mary Tawa",
+          "party": "People's"
+        },
+        {
+          "id": "rangel",
+          "name": "Damian Rangel",
+          "party": "Liberty"
+        },
+        {
+          "id": "altman",
+          "name": "Valarie Altman",
+          "party": "Constitution"
+        },
+        {
+          "id": "moore",
+          "name": "Helen Moore",
+          "party": "Constitution"
+        },
+        {
+          "id": "white",
+          "name": "John White",
+          "party": "Whig"
+        },
+        {
+          "id": "schmidt",
+          "name": "Joe Schmidt",
+          "party": "Whig"
+        },
+        {
+          "id": "smith",
+          "name": "Joe Smith",
+          "party": "Whig"
+        },
+        {
+          "id": "marracini",
+          "name": "Amanda Marracini",
+          "party": "Labor"
+        },
+        {
+          "id": "schreiner",
+          "name": "Martin Schreiner",
+          "party": "Independent"
+        }
+      ],
+      "writeIns": true
+    },
+    {
+      "id": "registrar-of-wills",
+      "type": "plurality",
+      "section": "County: 4th seat",
+      "title": "State Assembly",
+      "candidates": [
+        {
+          "id": "ramachandrani",
+          "name": "Rhadka Ramachandrani",
+          "party": "Independent"
+        }
+      ],
+      "writeIns": true
+    },
```

Comment:
not an inconsistency, I made `seats` optional. I think that's the right call rather than burden the data model with a field when it's most often `=1`.

Reply:
after discussion, I am convinced, will change it.

Reply:
after discussion, agreeing on `seats` as mandatory. 

Moving `yup` discussion to #98